client/src/hooks/useSearchResources.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useSearch } from './useSearch'
+import { ResourcesPortalContext } from '../ResouresPortalContext'
+
+export const useSearchResources = (newSearch) => {
+  const { search, setSearch } = React.useContext(ResourcesPortalContext)
+  // Should only be populated by results from get initial props
+  if (newSearch) Object.assign(search, newSearch)
+
+  const baseSearch = useSearch(search, setSearch)
+
+  return {
+    ...baseSearch,
+    query: search.query,
+    response: search.response
+  }
+}
```

Comment:
What's the distinction between this hook and useSearch? As far as I can tell this just uses the context to attach query and response, but nothing is actually specific to Resources? Couldn't that just be done in useSearch as well? Or is this in anticipation that different searches will attach different queries and responses? But still, doesn't it ultimately depend on what the context provided to the hook is?

Reply:
Yeah this is basically the same implementation as any other search type endpoint but this one needs to connect to the application context to preserve the response... other searches can just be regular `useSearch` hooks to interact with setting the search query etc

Reply:
Got it.